django-stubs/db/models/__init__.pyi

```diff
@@ -109,16 +109,6 @@ from .manager import BaseManager as BaseManager, Manager as Manager
 
 from . import lookups as lookups
 
-from .aggregates import (
```

Comment:
This is required: https://github.com/django/django/blob/master/django/db/models/__init__.py#L3

Reply:
It is above already (a bit more complete).
flake8 reported duplicate imports there.